kubernetes/samples/charts/nginx/README.md

```diff
@@ -0,0 +1,184 @@
+# Install and configure NGINX
+
+## A step-by-step guide to install the NGINX operator
+See the official installation document at:
+- https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
+
+As a *demonstration*, the following are steps to install the NGINX operator using Helm 3 on a Linux OS.
+
+### 1. Add the ingress-nginx chart repository
+```
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+```
+Verify that the chart repository has been added.
+
+```
+$ helm search repo ingress-nginx
+NAME               CHART VERSION APP VERSION	DESCRIPTION
+ingress-nginx/ingress-nginx	2.12.0       	0.34.1     	Ingress controller for Kubernetes using NGINX a...
+```
+> **NOTE**: After updating the Helm repository, the NGINX version listed may be newer that the one appearing here. Please check with the NGINX site for the latest supported versions.
+
+### 2. Install the NGINX operator
```

Comment:
maybe use term "NGINX ingress controller" here:
### 2. Install the NGINX ingress controller

Reply:
done